obspy/io/seiscomp/sc3ml.py

```diff
@@ -890,14 +890,15 @@ def _read_float_var(elem, cls, unit=False, datum=False, additional_mapping={}):
         convert = float(elem)
     except:
         warnings.warn(
-            "'%s' could not be converted to a float. Will be skipped. Please "
-            "contact to report this issue." % etree.tostring(elem),
+            "Encountered a value '%s' which could not be converted to a "
+            "float. Will be skipped. Please contact to report this "
```

Comment:
Well, the whole routine should be refactored. Seems like a copy/paste from a routine working with etree elements at some point but now working on string representations of floats. I think @Jollyfant implemented this module, maybe he finds some time for cleaning up this routine at some point, but I think we should do that in a separate issue. This one is really fixing bugs and should not be held up.